src/scripts/helpers/zotonic_setup

```diff
@@ -62,9 +62,13 @@ function find_enabled_sites {
 export -f find_enabled_sites
 
 function find_config {
+    # Allow admin to override default behaviour
+    [ ! -z "$ZOTONIC_CONFIG_DIR" ] && { echo "${ZOTONIC_CONFIG_DIR}/$1"; return 0; }
```

Comment:
This check can now be removed from `find_config_arg` as that calls `find_config`.